js/AssessmentSet.js

```diff
@@ -151,9 +151,10 @@ export default class AssessmentSet extends ScoringSet {
   }
 
   /**
-   * @override
+   * @extends
```

Comment:
Please revert these to override or remove them entirely

Extends is a synonym of augments, which is for `class Child extends Parent`
https://jsdoc.app/tags-augments.html

Override is for function overrides on the Child but isn't explicitly required.
https://jsdoc.app/tags-override.html